webapp/index.html

```diff
@@ -58,14 +58,6 @@ <h1 class="display-3">Shipping Application</h1>
             </div>
             <p></p>
             <div class="row">
-                <div class="col col-lg-1">
```

Comment:
You are not supposed to do any changes at the front end. If any of this code is blocking you from moving forward with your backend implementation, please suit yourself to ask the team what should be done.